coreneuron/sim/multicore.cpp

```diff
@@ -107,7 +107,7 @@ void nrn_threads_create(int n) {
 
 void nrn_threads_free() {
     if (nrn_nthread) {
-        free_memory((void*)nrn_threads);
+        delete[] nrn_threads;
```

Comment:
Not created by me, but it seems to be the reason why the test failed with travis.